.pipelines/3p-e2e.yml

```diff
@@ -46,7 +46,7 @@ resources:
         - repository: 1P
           type: git
           name: IDDP/msal-javascript-1p
-          ref: master
+          ref: keyvault_cert_sample
```

Comment:
this change is made for testing against the corresponding 1P change. This file will be reverted before the PR is merged into dev.